### PR TITLE
Update application.md for spa type 

### PR DESCRIPTION
**Pull request by hamiltonha**

### CODE REVIEW

File: api-reference/beta/resources/application.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ This resource supports using [delta query](/graph/delta-query-overview) to track
...
 | web                        | [webApplication](webapplication.md)                            | Specifies settings for a web application.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-
+| spa                        | [spaApplication](spaapplication.md)                                                | Specifies settings for a single-page application. `spa` type redirect URIs notify the server to enable CORS and use the authorization code flow. |
```
Comment (CelesteDG) [inline api-reference/beta/resources/application.md:90]: Missing the .. to get to where the spapplication.md file is (in api-reference/v1.0/resources folder). Not familiar enough with Graph, but should the stuff in beta be linking to v1.0 resources files? Just checking.

File: api-reference/beta/api/application-update.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ In the request body, supply the values for relevant fields that should be update
...
 | web                     | [webApplication](../resources/webapplication.md)                            | Specifies settings for a web application.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| spa                     | [spaApplication](spaapplication.md)                                          | Specifies settings for a single-page application. `spa` type redirect URIs notify the server to enable CORS and use the authorization code flow. |
```
Comment (CelesteDG) [inline api-reference/beta/api/application-update.md:65]: Missing the .. to get to where the spapplication.md file is (in api-reference/v1.0/resources folder). Not familiar enough with Graph, but should the stuff in beta be linking to v1.0 resources files? Just checking.

File: api-reference/v1.0/api/application-update.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ In the request body, supply the values for relevant fields that should be update
...
 | web                     | [webApplication](../resources/webapplication.md)                            | Specifies settings for a web application.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| spa                     | [spaApplication](spaapplication.md)                                         | Specifies settings for a single-page application. `spa` type redirect URIs notify the server to enable CORS and use the authorization code flow.                  |
```
Comment (CelesteDG) [inline api-reference/v1.0/api/application-update.md:62]: Missing the ../resources/ part of the link for the spaapplication.md.

**Comment by CelesteDG** [comment]
@hamiltonha - For the API Doctor errors, please see this page: https://msazure.visualstudio.com/One/_wiki/wikis/Microsoft%20Graph%20Partners/66012/API-Doctor-validate-examples

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please add an entry to the changelog.md file and the DirectoryServices.json changelog file to mention that this property is now available in v1.0.

**Pull request closed without merging**